sal_interfaces/HVAC/HVAC_Commands.xml

```diff
@@ -156,53 +86,15 @@
   </SALCommand>
   <SALCommand>
     <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_crack02P02_enable</EFDB_Topic>
-    <Description>Enable command for the crack02P02 device.</Description>
+    <EFDB_Topic>HVAC_command_config_fancoils</EFDB_Topic>
```

Comment:
Please use configFancoils (or fancoilsConfig) for the topic name. Or at least check with Tiago if you want to use snake_case instead.

Reply:
I chose configFancoils and also modified the enable and disable commands to enableDevice and disableDevice.